content/en/tracing/profiling/search_profiles.md

```diff
@@ -0,0 +1,139 @@
+| Socket I/O               | Shows the time each method spent handling socket I/O.                                                                                                                                                                                                                                              |
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU         | Shows the time each function spent running on the CPU. It includes CPython bytecode, including native code called from within Python.                                                                                                                                                                     |
+| Allocation               | Shows the amount of heap memory allocated by each function, including allocations which were subsequently freed. Only supported on Python 3.                                                                                                                                                                                    |
+| Allocation Count         | Shows the number of heap allocations made by each function, including allocations which were subsequently freed.     |
+| Wall Time | Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running. |
+| Exceptions               | Shows the number of caught or uncaught exceptions raised by each function.                                                                                           |
+| Lock                     | Shows the time each function spent in locking (waiting for or holding a lock) or the number of times a function was observed locking/unlocking a lock. |
+| Uncaught Exceptions      | Shows the exceptions that were not caught by any try/except block. |
+| Exceptions              | Shows the exceptions that were raised during program execution. |
```

Comment:
Dupe :-)

Reply:
I think they are two different types, no?

Reply:
Exceptions is listed twice.